### PR TITLE
MBS-9909: Change reports to use React (7 - ISWCs)

### PR DESCRIPTION
**Pull request by reosarevok**

### CODE REVIEW

File: root/report/ISWCsWithManyWorks.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import formatUserDate from '../utility/formatUserDate';
+import {l} from '../static/scripts/common/i18n';
+import PaginatedResults from '../components/PaginatedResults';
+import {WorkListRow} from '../static/scripts/common/components/WorkListEntry';
+
+import FilterLink from './FilterLink';
+import type {ReportDataT, ReportIswcT} from './types';
+
+
+const ISWCsWithManyWorks = ({
```
Comment (mwiencek) [inline root/report/ISWCsWithManyWorks.js]: There's some variable names with `ISWC` that I think we ought to rename to `Iswc` for consistency. Also in the actual file name.
```suggestion
const IswcsWithManyWorks = ({
```

File: root/report/ISWCsWithManyWorks.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  $c,
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportIswcT>) => {
+  let lastISWC = 0;
```
Comment (mwiencek) [inline root/report/ISWCsWithManyWorks.js]:
```suggestion
  let lastIswc = 0;
```

File: root/report/ISWCsWithManyWorks.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  let currentISWC = 0;
```
Comment (mwiencek) [inline root/report/ISWCsWithManyWorks.js]:
```suggestion
  let currentIswc = 0;
```

File: root/static/scripts/common/components/WorkListEntry.js [hunk 4]
```diff
@@ -19,22 +18,26 @@ import ArtistRoles from './ArtistRoles';
 import CodeLink from './CodeLink';
 import EntityLink from './EntityLink';
 
-type Props = {|
+type WorkListRowProps = {|
+  +hasISWCColumn: boolean,
```
Comment (mwiencek) [inline root/static/scripts/common/components/WorkListEntry.js]:
```suggestion
  +hasIswcColumn: boolean,
```

File: root/report/ISWCsWithManyWorks.js [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+              lastISWC = currentISWC;
+              currentISWC = item.iswc;
+
+              return (
+                <>
+                  {lastISWC !== item.iswc ? (
+                    <tr className="even" key={item.iswc}>
+                      <td>
+                        <a href={'/iswc/' + item.iswc}>{item.iswc}</a>
+                        <span>{' (' + item.workcount + ')'}</span>
```
Comment (mwiencek) [inline root/report/ISWCsWithManyWorks.js]: bracketed?

**Comment by reosarevok** [comment]
Since I was fixing style things, I also made a few stylistical fixes in the first merged Report files to make them follow the same style as the new ones. This is probably easier to check commit by commit.

**Comment by yvanzo** [comment]
```
/musicbrainz-server/root/report/CollaborationRelationships.js
  67:20  warning  Unexpected negated condition  no-negated-condition

/musicbrainz-server/root/report/DuplicateArtists.js
  79:22  warning  Unexpected negated condition  no-negated-condition

✖ 2 problems (0 errors, 2 warnings)
```

**Comment by reosarevok** [comment]
Those two are unrelated to this PR - would you prefer me to send an extra small commit here or a small commit on a separate PR? :)

**Pull request merged**